// [exit] 2
// [expect-error] Scanner error at line 3: Unexpected character.
foo(a | b)
